File: sandbox/node/src/TemplateListDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TemplateApi();

const accountId = undefined;
const page = 1;
const pageSize = 20;
const query = undefined;

apiCaller.templateList(
    accountId,
    page,
    pageSize,
    query,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Template#templateList:");
  console.log(error.body);
});

File: sandbox/ruby/src/TemplateListDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::TemplateApi.new.template_list(
        {
            account_id: nil,
            page: 1,
            page_size: 20,
            query: nil,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Template#template_list: #{e}"
end
